File: app/src/main/java/dk/itu/trashapp/MainActivity.java
package dk.itu.trashapp;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ItemsDB itemsDB = new ItemsDB();
        itemsDB.fillItemsDB();

        TextView items = findViewById(R.id.items);
        EditText inputText = findViewById(R.id.edit_text);
        Button listItems = findViewById(R.id.where_button);
        listItems.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                items.setBackgroundColor(Color.parseColor("#FFFFFF"));
                String a = "Trash List:" + "\n" + itemsDB.search(inputText.getText().toString().trim().toLowerCase());
                items.setText(a);
            }
        });
    }
}

File: app/src/main/java/dk/itu/trashapp/MainActivity.kt
package dk.itu.trashapp

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment


class MainActivity : AppCompatActivity() {

    private lateinit var itemsDB: ItemsDB

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ItemsDB.setContext(this@MainActivity)
        itemsDB = ItemsDB.get()
        setUpFragments()
    }

    private fun setUpFragments(){
        val fm = supportFragmentManager
        var fragmentUI : Fragment? = fm.findFragmentById(R.id.container_ui)
        var fragmentList : Fragment? = fm.findFragmentById(R.id.container_list)
        if(fragmentUI == null && fragmentList == null) {
            fragmentUI = SearchUIFragment()
            fragmentList = ListFragment()
            fm.beginTransaction()
                .add(R.id.container_ui, fragmentUI)
                .add(R.id.container_list, fragmentList)
                .commit()
        }
    }
}